src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -113,13 +123,24 @@ class SceneryWindow final : public Window
 
     struct SceneryTabInfo
     {
+        SceneryTabType Type = SCENERY_TAB_TYPE_GROUP;
         ObjectEntryIndex SceneryGroupIndex = OBJECT_ENTRY_INDEX_NULL;
         std::deque<ScenerySelection> Entries;
```

Comment:
Default initializing this should fix the new warnings:
```suggestion
        std::deque<ScenerySelection> Entries{};
```

Reply:
Hopefully all the builds pass now.